docs/chaos/latency.md

```diff
@@ -90,15 +92,45 @@ var pipeline = new ResiliencePipelineBuilder()
 
 ## Defaults
 
-| Property            | Default Value | Description                                            |
-|---------------------|---------------|--------------------------------------------------------|
-| `Latency`           | `30 seconds`  | A `TimeSpan` indicating the delay to be injected.      |
-| `LatencyGenerator`  | `null`        | Generates the latency to inject for a given execution. |
-| `OnLatencyInjected` | `null`        | Action executed when latency is injected.              |
+| Property            | Default Value | Description                                                                                                          |
+|---------------------|---------------|----------------------------------------------------------------------------------------------------------------------|
+| `Latency`           | 30 seconds    | Defines a **static** delay to be injected.                                                                           |
```

Comment:
Fixed or constant to avoid confusion with `static`?

Reply:
The same wording is used in case of [Circuit Breaker](https://www.pollydocs.org/strategies/circuit-breaker.html#defaults) and [Timeout](https://www.pollydocs.org/strategies/timeout.html#defaults) as well.

Shall I rename there as well?

Reply:
Done